core/tests/widgets/test_container_debug_mode.py

```diff
@@ -0,0 +1,57 @@
+from os import environ
+
+from travertino.colors import rgb
+
+import toga
+
+
+# test that a non-container-like widget in layout debug mode has a default background
+def test_button_debug_background():
```

Comment:
Good idea testing a widget other than Box; - but also worth adding at test for (a) a container widget, and (b) a widget where the debug background *won't* be applied.